packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md

```diff
@@ -0,0 +1,175 @@
+
+## Before you begin
+
+This guide assumes that you:
+
+* Have an Okta Developer Edition organiztion. Don't have one? [Create one for free](https://developer.okta.com/signup).
+* Have the Native SSO feature enabled for your org. From the left navigation pane in the Admin Console, go to **Settings** > **Features**, locate the Native SSO slider, and slide to enable.
+
+## Configure an application to use the Device Authorization Grant
+
+To create a Native OpenID Connect application and then configure it to support the Device Authorization Grant type:
+
+1. In the left navigation pane of the Admin Console, go to **Applications** > **Applications**.
+1. Click **Create App Integration**.
+1. On the Create a new app integration page, select **OIDC - OpenID Connect** as the **Sign-in method**, and then pick **Native Application**.
+1. On your native application page, fill in the application settings. Ensure that you select **Device Authorization** as the allowed **Grant type** in the General Settings.
+
+> **Note:** The Device Authorization Grant is only supported for use with a native application.
+
+## Configure the Authorization Server policy rule for the Device Authorization grant type
+
+Both Org and Custom Authorization Servers support the Device Authorization grant type. Ensure that the Device Authorization grant type is enabled at the policy rule level.
+
+To check that the Device Authorization grant type is enabled:
+
+1. In the left navigation pane of the Admin Console, go to **Security** > **API** and select the "default" Custom Authorization Server. Note that the examples in this guide use the "default" Custom Authorization Server.
+1. On the **Access Policies** tab, select the access policy that you want to configure Device Authorization for.
+1. Click the pencil icon for the Default Policy Rule.
+1. In the Edit Rule dialog box, select **Device Authorization** for the grant type and click **Update Rule**.
+
+## Configure the smart device
+
+### Request the device verification code
+
+The smart device first needs to call the `/device/authorize` endpoint to obtain the unique verification code.
+
+**Request example**
+
+```bash
+  curl --request POST \
+  --url https://${yourOktaDomain}/oauth2/default/v1/device/authorize \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id={clientId}' \
+  --data-urlencode 'scope=openid profile offline_access'
+```
+
+The POST request passes the following parameters:
+
+* `client_id`, which matches the Client ID of the OAuth 2.0 application that you created
+* `scope` to specify which access privileges are being requested for access tokens. See [Scopes](https://developer.okta.com/docs/reference/api/oidc/#scopes) for a list of supported scopes.
+
+**Response example**
+
+```json
+{
+   "device_code": "f5ec0430-c2fe-4a93-912d-08631f1effed",
+   "user_code": "HMBSRJRC",
+   "verification_uri": "https://${yourOktaDomain}/activate",
```

Comment:
Are we only parameterizing the org domain but nothing else in the response like the device code and user code?

Reply:
In this example, yes, it's OK as is, as per the style guide and consistent use in the other guides.